### ISSUE TITLE
accessibility model view fails

### ISSUE THREAD

**Issue by smnorris** [smnorris/bcfishpass#383]
presumably the query has not been updated with the small changes to data model:
```
psql:sql/freshwater_fish_habitat_accessibility_model.sql:68: ERROR:  column s.remediated_dnstr does not exist
LINE 56:   s.remediated_dnstr,
```

**Comment by smnorris** [comment]
fixed with https://github.com/smnorris/bcfishpass/pull/385
